### PR TITLE
new functions

### PR DESCRIPTION
**Pull request by tommmyy**
Solves #61

### CODE REVIEW

File: packages/ramda-extension/src/padRightUntil.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import { until, flip, identity, concat, useWith } from 'ramda';
+
+/**
+ * Takes a predicate, string `padString` and initial value. `padString` is prepend to the output string
```
Comment (jbradle) [inline packages/ramda-extension/src/padRightUntil.js]: It is append

File: packages/ramda-extension/src/padRightUntil.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+ * everytime `pred` returns falsy value.
+ *
+ * @param {fn} pred Called after every prepend of `padString`.
```
Comment (jbradle) [inline packages/ramda-extension/src/padRightUntil.js]: Append

File: packages/ramda-extension/src/padLeftUntil.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
+import { until, identity, concat, useWith } from 'ramda';
+
+/**
+ * Takes a predicate, string `padString` and initial value. `padString` is contacted to the output string
+ * everytime `pred` returns falsy value.
+ *
+ * @param {fn} pred Called after every contactation of `padString`
```
Comment (jbradle) [inline packages/ramda-extension/src/padLeftUntil.js]: What is pred? :)

Comment (tommmyy) [reply packages/ramda-extension/src/padLeftUntil.js]: see names of the params bellow

**Comment by tommmyy** [comment]
@jbradle review checked

**Comment by jbradle** [comment]
merge it 👍

**Pull request merged**